pkg/server/sandbox_status.go

```diff
@@ -58,24 +58,50 @@ func (c *criContainerdService) PodSandboxStatus(ctx context.Context, r *runtime.
 			state = runtime.PodSandboxState_SANDBOX_READY
 		}
 	}
+
+	ip, err := c.getIP(sandbox)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get sandbox ip: %v", err)
+	}
+
+	createdAt := sandbox.Container.Info().CreatedAt
+	status := toCRISandboxStatus(sandbox.Metadata, state, createdAt, ip)
+	return &runtime.PodSandboxStatusResponse{Status: status}, nil
+}
+
+func (c *criContainerdService) getIP(sandbox sandboxstore.Sandbox) (string, error) {
 	config := sandbox.Config
+
+	if config.GetLinux().GetSecurityContext().GetNamespaceOptions().GetHostNetwork() {
+		// For sandboxes using the host network we are not
+		// responsible for reporting the IP.
+		glog.V(10).Infof("getIP: %s: sandbox is using host network", sandbox.ID)
```

Comment:
This is quite normal, maybe not log it?

Reply:
Yeah, I put it in to validate my code was working then wasn't sure whether to keep it or not, so I just lowered the severity. Will drop this and adjust the others as you suggest.